app/controllers/users.js

```diff
@@ -14,3 +17,27 @@ exports.create = (req, res, next) => {
     .then(user => res.status(201).send(user))
     .catch(reason => next(errors.defaultError(`Database error - ${reason}`)));
 };
+
+exports.authenticate = (req, res, next) => {
+  User.findOne({ where: { email: req.body.email } })
+    .then(user => {
+      const hashedPassword = encode().value(req.body.password);
+      if (user.password === `${hashedPassword}`) {
+        const token = jwt.sign({ user_id: user.id }, JWT_KEY),
+          userWithToken = {
+            email: user.email,
+            password: user.password,
+            first_name: user.firstName,
+            last_name: user.lastName,
+            session_token: token,
+            created_at: user.created_at,
+            updated_at: user.updated_at
+          };
+        res.status(201).send(userWithToken);
+      } else {
+        console.log(`Error passwords: DB: ${user.password} VS Req: ${hashedPassword}`);
```

Comment:
delete console log, use logger

Reply:
Is this really necessary now? Maybe we can not include this for now